app/src/main/java/org/sagebionetworks/research/mpower/tracking/fragment/MedicationDayFragment.kt

```diff
@@ -57,19 +57,19 @@ class MedicationDayFragment : AppCompatDialogFragment() {
         var days : String? = null
         if (savedInstanceState == null) {
             if (arguments != null) {
-                schedId = arguments!!.getString(ARG_SCHED_ID)
-                name = arguments!!.getString(ARG_NAME)
-                time = arguments!!.getString(ARG_TIME)
+                schedId = arguments!!.getString(ARG_SCHED_ID)!!
```

Comment:
You don't want to be a [NPE-lover](https://kotlinlang.org/docs/reference/null-safety.html#the--operator) do you!?

Even Kotlin docs joke about using "!!" because it doesn't really make any sense. Check out the "let" extension function, it will take care of all these "!!". 

For instance...

`arguments?.getString(ARG_SCHED_ID)?.let { schemed = it }`